.vscode/settings.json

```diff
@@ -7,12 +7,9 @@
 		"tgui/.yarn": true,
 		"tgui/.pnp.*": true
 	},
-	"workbench.editorAssociations": [
```

Comment:
Why was it changed?

Reply:
ops, sorry, in teory i discarded that, i will change it now

Reply:
In teory now it works well, it seems like visual studio code made like that when I opened it